### PR TITLE
MAUI-1459-Updated UG review correction for SfScheduler.

### PR DESCRIPTION
**Pull request by Karthickmani97**
Updated UG review correction for SfScheduler.

### CODE REVIEW

**Review by Jeyasri-Murugan — APPROVED** [review]

File: MAUI/Scheduler/appointments.md [hunk 1]
```diff
@@ -518,19 +518,19 @@ var date3 = 12/16/2021 10:00:00 AM;
 
 The Recurrence pattern appointments can be deleted or changed by handling exception dates and exception appointments.
 
-### Recurrence exception dates
+#### Recurrence exception dates
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/appointments.md:521]: ?

File: MAUI/Scheduler/appointments.md [hunk 2]
```diff
@@ -518,19 +518,19 @@ var date3 = 12/16/2021 10:00:00 AM;
...
 
 By adding exception dates to the recurring appointment, any occurrence appointment can be removed that is an exception from the recurrence pattern.
 
-### Recurrence exception appointment
+#### Recurrence exception appointment
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/appointments.md:525]: ?

File: MAUI/Scheduler/appointments.md [hunk 3]
```diff
@@ -824,11 +824,11 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/appointments.md:825]: Add an exception appointment to the recurrence pattern - custom app - make it H4

File: MAUI/Scheduler/load-on-demand.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ These can be achieved in two ways:
 * QueryAppointments event
 * QueryAppointments command
 
-### QueryAppointments event
+#### QueryAppointments event
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/load-on-demand.md:25]: ?

File: MAUI/Scheduler/load-on-demand.md [hunk 5]
```diff
@@ -125,7 +125,7 @@ N> The QueryAppointments event will be triggered, if any one of the following ac
 * In the scheduler agenda view, the `QueryAppointments` event is used to load more appointments when the new month is loaded on view, whereas the new month appointments are added in scheduler `AppointmentsSource.`
 * Other than agenda view, the scheduler `AppointmentsSource` can be reset for a new visible date range to improve appointment loading performance.
 
-### QueryAppointments command
+#### QueryAppointments command
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/load-on-demand.md:128]: ?

File: MAUI/Scheduler/load-on-demand.md [hunk 6]
```diff
@@ -286,7 +286,7 @@ N>
 * When using the `QueryAppointments` or `QueryAppointmentsCommand,` you can start and stop the animation before and after the appointments load.
 * When `QueryAppointments` or `QueryAppointmentsCommand` are not used, and show busy indicator is enabled, the busy indicator will be stopped once the appointment is rendered.
 
-## Busy indicator appearance
+#### Busy indicator appearance
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/load-on-demand.md:289]: H2

**Pull request merged**